content/pages/sponsorships.md

```diff
@@ -0,0 +1,48 @@
+
+<h2>Baseline package</h2>
+
+Any company willing to contribute can get involved choosing from different levels of sponsorship and perks. As a baseline we offer a 1000 EUR sponsorship, which includes:
+
+* your company logo on the website and in the intro and closing slides of each block
+* your company is mentioned by our hosts during the conference
+* 10 tickets to hand out to your own employees, or anyone else
+
+<h2>Tailored to your needs</h2>
+
+We are always willing to discuss more elaborate sponsorships as well. To inspire you we have prepared this non-exhaustive list of potential perks:
+
+* <b>Unique web experiences during the conference</b>
+
+    RustFest Global will be live streamed on our website. The page that hosts the live stream can be customized with passive (e.g. design) and active (e.g. interactive) content. Examples may include a cute Ferris game in Rust and/or Webassembly. However we do require that this content is hosted by RustFest and that the content respects the privacy of our attendees. Just like a rollup banner, this web experience is either provided by you as the sponsor or commissioned by the sponsor to be created by the Rustfest organizing team.
+
+* <b>Unique web experiences on the schedule page</b>
+
+    The second most visited page of our conference website has always been the schedule page, which is where people go to re-watch previous talks. This experience can also be customized to include sponsored content.
+
+* <b>Myriad of things in Matrix!</b>
+
+    We will use Matrix rooms for our chat system during the conference. Our streaming service <i>Waasabi</i> lets us tightly integrate Matrix into the stream page. This means that the chat room will be right under the actively streaming video. As a sponsor you could have a Matrix bot, or a special sponsor chat room on Matrix.
+
+* <b>In-break content</b>
+
+    During breaks we can include sponsorship ads that target our attendees. Like with the similar Oxidize conference we also include generative art and music from artist to play during the breaks. This generative art could also be customized to your brand, using your color scheme or include some of your text. We can also discuss a direct collaboration between you and the artist.
+
+* <b>Playback content</b>
+
+    All RustFest talks will also be available to watch for free on YouTube some time after the conference. Our talks from RustFest Barcelona have been viewed between 2000 and 5000 times on these platforms. We can include an intro or outro video advertisement with these videos.
```

Comment:
Maybe let's be more vague and just talk about all rustfest videos (some of our best talks reached 30k+ views)

Reply:
Our talks from RustFest have been watched thousand of times... nahh

Our talks from past RustFests still receive thousands of views.

Reply:
Reworked this bit.